### PR TITLE
Update PUP3 with rel-eng changes

### PR DESCRIPTION
**Pull request by pcreech**

### CODE REVIEW

File: pup-0003.md [hunk 1]
```diff
@@ -76,10 +76,12 @@ feature, bug fix, or hotfix.
 
 1. All bug fix PRs are opened against master
 2. After the PR is ready to be merged, the person merging the commit would
-   first merge the commit to master. Then, this person cherry-picks this change
-   from master to the latest x.y-dev branch by opening a PR with the change.
-3. The person cherry-picking would also need to track in Redmine which dev
-   branch(es) the change has been applied to.
+   first merge the commit to master. Then, this person updates the platform
+   relesae field on the issue.  Automated tooling will, at release time,
```
Comment (daviddavis) [inline pup-0003.md]: s/relesae/release/

File: pup-0003.md [hunk 2]
```diff
@@ -76,10 +76,12 @@ feature, bug fix, or hotfix.
...
+   attempt to cherry-pick from master to the latest x.y-release branch
+   opening a PR with all the changes for a release.
+3. The person fixing the issue would also need to track in Redmine which
+   release the change is tied to.
```
Comment (daviddavis) [inline pup-0003.md]: Is this the same as the previous step (ie "updating the platform release field on the issue")?

File: pup-0003.md [hunk 3]
```diff
@@ -90,14 +92,15 @@ instead of merging forward the commit, we'd use cherry-picking.
...
-3. The change is then cherry-picked into the x.y-dev branch and master by PR.
+3. The change is then cherry-picked into master by PR.
 4. The person cherry-picking would also need to track in Redmine which release
    branches the change has been made to.
 
 ### Conflicts
 
-If the cherry-pick has conflicts, we should abandon the cherry-pick and
-encourage users to upgrade to the next Y release.
+If the cherry-pick has conflicts, we should asses why the conflicts arose (missed
```
Comment (daviddavis) [inline pup-0003.md]: assess

File: pup-0003.md (hunk 2 above)
Comment (pcreech) [reply pup-0003.md]: Sure is, i was updating tranactionally it seems

File: pup-0003.md (hunk 3 above)
Comment (pcreech) [reply pup-0003.md]: fixed

File: pup-0003.md (hunk 1 above)
Comment (pcreech) [reply pup-0003.md]: fixed

File: pup-0003.md [hunk 4]
```diff
@@ -47,7 +47,7 @@ newer ones) still need to be merged forward. We could elminate the need for
 this with cherry-picking. It's worth noting though that this rarely happens.
```
Comment (ammaritiz) [inline pup-0003.md:47]: Previous line. s/elminate/eliminate

Comment (pcreech) [reply pup-0003.md:47]: @ammaritiz edited, nice catch!

**Comment by daviddavis** [comment]
LGTM. Once this gets a second approval, will merge.

**Review by dralley — APPROVED** [review]

**Review by daviddavis — APPROVED** [review]

**Pull request merged**